Bottom: 20772bd3b9dc28a5f87eddc9de6a92ffa56261c0
Top:    edfa0eb42e5d80dc42f7676d6892c8657c51ebb0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:01:13 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
